webxdc.js

```diff
@@ -1 +1,285 @@
-/* dummy file so bundler can be happy - see https://github.com/webxdc/webxdc-types/issues/5 */
\ No newline at end of file
+// This file originates from
+// https://github.com/webxdc/hello/blob/master/webxdc.js
+// It's a stub `webxdc.js` that adds a webxdc API stub for easy testing in
+// browsers. In an actual webxdc environment (e.g. Delta Chat messenger) this
+// file is not used and will automatically be replaced with a real one.
+// See https://docs.webxdc.org/spec.html#webxdc-api
```

Comment:
this addition does not relate to the PR topic.  Why not as a PR to the hello-repo which would diverge as soon as we merge this PR here?

Reply:
Hello repo also has a PR. For people who only look at this repo it might be interesting to have a mock implementation as well. The true source is linked so a  divergence is acceptable imo.